ui/src/components/thumbnail/thumbnail.tsx

```diff
@@ -144,48 +195,69 @@ export const Thumbnail = ({
           style={{ display: 'none' }}
           {...props}
         />
-      </div>
-      <div className="thumbnail-actions">
-        <Button
-          id="view-thumbnail-btn"
-          variant="contained"
-          color="secondary"
-          size="small"
-          startIcon={<VisibilityIcon />}
-          onClick={handleViewThumbnail}
-          disabled={!currentFile && !currentImage}
+      </Box>
+      <Box sx={{ display: 'flex', flexDirection: 'column', rowGap: '16px' }}>
+        <Box
+          sx={{
+            display: 'flex',
+            flexDirection: { xs: 'column', sm: 'row' },
+            columnGap: '16px',
+            rowGap: '12px',
+            height: '40px',
+          }}
         >
-          View Thumbnail
-        </Button>
-        <Button
-          id="upload-thumbnail-btn"
-          variant="contained"
-          color="secondary"
-          size="small"
-          startIcon={<UploadFileIcon />}
-          onClick={handleBrowseThumbnails}
-        >
-          Upload Thumbnail
-        </Button>
-        <Button
-          id="remove-thumbnail-btn"
-          variant="contained"
-          color="secondary"
-          size="small"
-          startIcon={<DeleteIcon />}
-          onClick={handleRemoveThumbnail}
-          disabled={!currentFile && !currentImage}
+          <Button
+            id="upload-thumbnail-btn"
+            variant="contained"
+            color="secondary"
+            startIcon={<UploadRoundedIcon />}
+            onClick={handleBrowseThumbnails}
+            sx={{ width: '100%', maxWidth: '170px' }}
+          >
+            Select an image
+          </Button>
+          {currentFile || currentImage ? (
+            <Button
+              id="remove-thumbnail-btn"
+              variant="contained"
+              color="secondary"
+              startIcon={<DeleteIcon />}
+              onClick={handleRemoveThumbnail}
+              disabled={!currentFile && !currentImage}
+              hidden={!currentFile && !currentImage}
```

Comment:
This seems redundant based on the conditional in which the  image will be rendered or not if currentFile || currentImage